spec/unit/mutant/subject/method/instance_spec.rb

```diff
@@ -81,10 +81,6 @@ def self.name
   let(:context)  { Mutant::Context.new(scope, double('Source Path')) }
   let(:node)     { Unparser.parse('def foo; end')                    }
 
-  before do
-    allow(Object).to receive_messages(const_get: scope)
-  end
-
```

Comment:
@mbj Can you help me understand why this was added originally?

Reply:
I suspect this is very old. Before mutant started to use the `Mutant::World` to incrementally phase out this entire concept of "mocking IO globals".

Reply:
nope, its me adding it. I must have been at sleep at the wheel.

Reply:
I suspect that this was a temporary change to test something that leaded into whats committed unintentionally.